manifests/dev/libvirt.yaml.in

```diff
@@ -12,7 +12,7 @@ spec:
       labels:
         kubevirt.io: libvirt
     spec:
-      serviceAccountName: kubevirt-infra
+      serviceAccountName: kubevirt-privileged
```

Comment:
I don't think that the libvirt pod ever has to talk to the cluster. Are you sure it needs a privileged account?

Reply:
- Do you refer to the `cluster-admin` role that was added to `kubevirt-privileged`?
- The name `kubevirt-privileged` indicates that this SA has a privileged scc (when working with 
   Openshift)

Reply:
> Do you refer to the cluster-admin role that was added to kubevirt-privileged?

I think that the libvirt pod does not need a service account at all.

> The name kubevirt-privileged indicates that this SA has a privileged scc (when working with
> Openshift)

I don't mind the prefix. We can also keep it, but virt-handler should actually not need cluser admin provileges, it is just that no one went through all resources it can touch and create a special role for it. We can sort that out in another PR.

Reply:
The service account on libvirt is needed on openshift deployments.
On openshift deployments an SCC is associated with this SA in order to give it the right (node level) privileges.

I agree that we should fix/differentiate cluster wide and node level wide privileges in a follow up pr.

Reply:
good to know thx.